lambdas/handlers/authoriser_handler.py

```diff
@@ -33,20 +37,26 @@ def lambda_handler(event, context):
         )
         public_key = ssm_response["Parameter"]["Value"]
 
-        # For now we just assume the user authenticate with a signed token in this form:
-        # { "user_role": "Unauthorised"  or  "GP"  or  "PCSE" }
-        # The token is going to be issued by the lambda of ticket PRMDR-167
-
         decoded = jwt.decode(
             event["authorizationToken"], public_key, algorithms=["RS256"]
         )
-        user = decoded["user_role"]
-        logger.info(f"decoded JWT: {decoded}")
+        logger.debug(f"decoded JWT auth token: {decoded}")
```

Comment:
Are we planning on removing this?

Reply:
Seems a bit naughty to log all JWT tokens, maybe both should be for debug mode.

Reply:
Agree. Logging this token was just for dev purpose, so now this logging is removed.